Sources/Foundation/Process.swift

```diff
@@ -653,6 +653,7 @@ open class Process: NSObject, @unchecked Sendable {
                 process._terminationReason = .exit
             }
 
+            CloseHandle(process.processHandle)
```

Comment:
You might want to instead do this within terminateRunLoop after the termination handler is called (or at the end of terminateRunLoop if there is no termination handler). Seems safest to do this as late as possible.